packages/seacas/applications/exodiff/ED_Version.h

```diff
@@ -30,10 +30,10 @@
 // (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 //
-#ifndef SEACAS_Version_h
-#define SEACAS_Version_h
+#ifndef ED_Version_h
+#define ED_Version_h
 
-static std::string version("2.92");
-static std::string verdate("2018-06-27");
+static std::string version("2.94");
```

Comment:
I'm not a fan of `static std::string`, though I suppose it's OK.  Eventual C++ plan is to "`constexpr` _ALL_ THE THINGS!!!!1111!1!!1!11ONE!" but what's wrong with `const char version[] = "2.94";` for now?